### ISSUE TITLE
Typing issue with the latest version of minipass

### ISSUE THREAD

**Issue by Stono** [isaacs/minipass#40]
Hey, 
We use https://www.npmjs.com/package/tar which in turn uses minipass, compiling against the latest version of minipass (and using @types/minipass) gives the following error:

```
#24 9.278 node_modules/@types/tar/index.d.ts(252,36): error TS2709: Cannot use namespace 'MiniPass' as a type.
```

Pinning `minipass` to `3.2.1` in our project works around it.

Wasn't sure where best to raise this, i'll cross-raise it on the tar repo too!

**Comment by nex3** [comment]
I believe the root issue here is https://github.com/DefinitelyTyped/DefinitelyTyped/discussions/60901.

**Comment by MichalCz** [comment]
We have a similar issue in Transform Hub... https://github.com/scramjetorg/transform-hub/runs/7006507844?check_suite_focus=true.

**Comment by MysteryBlokHed** [comment]
I think this is caused because of the bundled types using a different kind of export than the current ones on DefinitelyTyped.
I opened #41 a little while ago which I think should fix this

**Comment by isaacs** [comment]
Closed by #41
